lib/dependents/monitor.rb

```diff
@@ -38,7 +38,7 @@ def track_submission_exhaustion(msg, email = nil)
 
       StatsD.increment("#{SUBMISSION_STATS_KEY}.exhausted")
       Rails.logger.error(
-        "Failed all retries on CentralMail::SubmitCentralForm686cJob, last error: #{msg['error_message']}"
+        "Failed all retries on Lighthouse::BenefitsIntake::SubmitCentralForm686cJob, last error: #{msg['error_message']}" # rubocop:disable Layout/LineLength
```

Comment:
Here's another line that could be split up to avoid skirting the rubocop check

Reply:
This has been resolved in my latest push @ericboehs